src/ass/kfx.py

```diff
@@ -0,0 +1,237 @@
+from collections.abc import Sequence, Mapping
+
+from datetime import timedelta
+from pyass import Color
+from pyass import Alignment
+import pyass.tag
+
+from .consts import *
+from ..models.karaoke import *
+
+
+def get_line_format(line: KLine) -> pyass.EventFormat:
+    return (
+        pyass.EventFormat.COMMENT
+        if line.isAlone and line.isEN
+        else pyass.EventFormat.DIALOGUE
+    )
+
+
+def get_romaji_pos_tag(line: KLine) -> pyass.Tag:
+    if line.isSecondary:
+        return SECONDARY_ROMAJI_POS_TAG
+    elif line.isAlone:
+        return NO_EN_ROMAJI_POS_TAG
```

Comment:
Keeping with the terminology we could rename this to "ALONE_ROMAJI_POS_TAG" or similar

Reply:
Sure, that works